INSTRUCTION.md

```diff
@@ -0,0 +1,4 @@
+kind create cluster --config cluster.yml
+kubectl apply -f ingress.yml
```

Comment:
Ensure that the path 'ingress.yml' is correct. If the ingress.yml file is located in a subdirectory (e.g., .infrastructure/ingress/), you should provide the relative path to it, such as '.infrastructure/ingress/ingress.yml'.